uber/templates/regform.html

```diff
@@ -443,7 +443,7 @@ <h4 class="h4-inline">
     </div>
 {% else %}
     <div class="form-group">
-        <label for="zip_code" class="col-sm-2 control-label">ZIP/Postal Code</label>
+        <label for="zip_code" class="col-sm-2 control-label {% if c.PAGE_PATH == '/registration/register' %}optional-field{% endif %}">ZIP/Postal Code</label>
```

Comment:
The actual model check which makes this optional for this page on the backend is the line in ``model_checks.py``

```
if not attendee.international and not c.AT_OR_POST_CON:
```

I think it would be therefore better to say

```
{% if c.AT_OR_POST_CON %}optional-field{% endif %}
```

since then it'll match the actual condition we're checking on.